### PR TITLE
Cron

### PR DESCRIPTION
**Pull request by gonzalezcesar**

### CODE REVIEW

File: src/main/webapp/WEB-INF/cron.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cronentries>
+    <cron>
+        <url>/cron</url>
+        <description>Update subscribers about new posts(if any)!</description>
+        <!-- for testing cron job -->
+        <schedule>every 2 minutes from 17:51 to 17:53</schedule>
```
Comment (JohnStarich) [inline src/main/webapp/WEB-INF/cron.xml]: Perhaps this schedule should read `every day`?

Comment (JohnStarich) [inline src/main/webapp/WEB-INF/cron.xml]: Go ahead and change this to `every day` like the comment below when we create pull requests to master. That way when we deploy master its always the correct interval.

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -12,6 +12,15 @@
 		<servlet-name>login</servlet-name>
 		<servlet-class>com.johnstarich.ee461l.hw1.LoginServlet</servlet-class>
 	</servlet>
+	<servlet>
+		<servlet-name>cronjob</servlet-name>
+		<servlet-class>com.johnstarich.ee461l.hw1.CronServlet</servlet-class>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>cronjob</servlet-name>
+		<!-- need to agree on URL for cron -->
```
Comment (JohnStarich) [inline src/main/webapp/WEB-INF/web.xml]: `/cron` is fine :+1:

File: src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -20,6 +29,17 @@
 		<servlet-name>posts</servlet-name>
 		<url-pattern>/api/posts/*</url-pattern>
 	</servlet-mapping>
+	<!-- web-resource-name? what is it? -->
```
Comment (JohnStarich) [inline src/main/webapp/WEB-INF/web.xml]: We resource name is whatever you like, so `cron` is fine

**Comment by JohnStarich** [comment]
LGTM :+1: merge when you've reviewed my comments

**Comment by gonzalezcesar** [comment]
cool. will do. I'll schedule the cron job for every 24 hours and push the change.

**Pull request merged**